### PR TITLE
9.0.1.33978 Updates

### PR DESCRIPTION
**Pull request by funjoker**

### CODE REVIEW

**Comment by funjoker** [comment]
Seems like no need

**Comment by bloerwald** [comment]
Hi,
I started working on it yesterday without keeping an eye on merge requests, so I completely missed all your work until I went to bed. Once I continue today, I’ll surely be glad about those files you already did and pick those. 

Sorry again for missing it!

**Comment by funjoker** [comment]
Let me do a rebase first now

**Comment by funjoker** [comment]
In gameTips there are twice the same field defined

**Comment by funjoker** [comment]
done for now (work)

File: definitions/QuestPOIBlob.dbd [hunk 1]
```diff
@@ -8,14 +8,7 @@ int ObjectiveIndex
...
 int<UiMap::ID> UiMapID
-int Field_9_0_1_33978_000?
-int Field_9_0_1_33978_001?
-int Field_9_0_1_33978_002?
-int Field_9_0_1_33978_003?
-int Field_9_0_1_33978_004?
-int Field_9_0_1_33978_005?
-int Field_9_0_1_33978_006?
-int Field_9_0_1_33978_007?
+int ObjectiveID
```
Comment (funjoker) [inline definitions/QuestPOIBlob.dbd:11]: here marlamin can do some linking soon i guess

File: definitions/QuestPOIPoint.dbd [hunk 2]
```diff
@@ -37,8 +33,8 @@ $noninline,relation$QuestPOIBlobID<32>
 
 LAYOUT D291971B
 BUILD 9.0.1.33978
-$id$Field_9_0_1_33978_000<32>
-Field_9_0_1_33978_001<16>
-Field_9_0_1_33978_002<16>
+$id$ID<32>
+X<16>
+Y<16>
 Field_9_0_1_33978_003<16>
```
Comment (funjoker) [inline definitions/QuestPOIPoint.dbd:39]: looks somewhat Z like

**Comment by Marlamin** [comment]
Gonna merge this as-is now and will likely take-over for a bit (until something else comes up) with FK mappings and such. Thanks a lot!

**Pull request merged**